example/19_bnorm_fwd/bnorm_fwd_nhwc_blockwise.cpp

```diff
@@ -0,0 +1,372 @@
+#include <iostream>
+#include <fstream>
+#include <numeric>
+#include <initializer_list>
+#include <cstdlib>
+#include <getopt.h>
+#include "check_err.hpp"
+#include "config.hpp"
+#include "print.hpp"
+#include "device.hpp"
+#include "host_tensor.hpp"
+#include "host_tensor_generator.hpp"
+#include "host_common_util.hpp"
+#include "device_tensor.hpp"
+#include "reference_bnorm_fwd_nhwc_c.hpp"
+#include "device_bnorm_fwd_nhwc_c_with_reduce_blockwise.hpp"
+
+using namespace ck;
+using namespace ck::tensor_operation::device;
+using namespace ck::tensor_operation::host;
+
+using InOutDataType = ck::half_t;
+using AccDataType   = float;
+
+static const double exponentialAverageFactor = 0.2;
+static const double epsilon                  = std::numeric_limits<AccDataType>::epsilon();
+
+using ReferenceBatchNormFwdInstance =
+    ReferenceBatchNormFwd_Input_N_H_W_C_Output_C<InOutDataType, AccDataType>;
+
+using DeviceBatchNormFwdInstance = DeviceBatchNormFwd_Input_N_H_W_C_Output_C_With_Reduce_Blockwise<
+    InOutDataType,
+    AccDataType,
+    256, // BlockSize
+    8,   // MThreadClusterSize,
+    32,  // KThreadClusterSize,
+    2,   // MThreadSliceSize,
+    1,   // KThreadSliceSize,
+    2,   // InOutVectorSize,
+    2>;  // ScaleBiasMeanVarVectorSize
+
+static struct option long_options[] = {{"inOutLengths", required_argument, nullptr, 'D'},
+                                       {"verify", required_argument, nullptr, 'v'},
+                                       {"help", no_argument, nullptr, '?'},
+                                       {nullptr, 0, nullptr, 0}};
+
+class SimpleAppArg
```

Comment:
example of bnorm_fwd_nhwc_blockwise/multiblock share a lot of code, could we extract common codes into a head file?

Reply:
The same/similar names,  but different codes in different cpp files